packages/cms-editor/src/main/control/language-tool/LanguageTool.tsx

```diff
@@ -125,6 +126,11 @@ export const LanguageTool = () => {
         locales?.filter(locale => !args.locales.includes(locale))
       );
       client.removeLocales({ context, locales: args.locales });
+    },
+    onSuccess: () => {
+      if (JSON.stringify(initialDefaultLanguages().sort()) === JSON.stringify(defaultLanguages.sort())) {
```

Comment:
instead of JSON.stringify, you can also use deepEquals or equalsDeep of the ui-comonents